misc/python/materialize/checks/multiple_partitions.py

```diff
@@ -101,7 +101,7 @@ def validate(self) -> Testdrive:
             dedent(
                 """
                 > SELECT partition FROM multiple_partitions_source_progress;
-                (3,)
+                \\(3,)
```

Comment:
<strike>How is this related to the changes?</strike> I understand that we now need to escape normal usages of parentheses.

Does the version specification always combe before one of the `$`, `>`, `?` chars? If so, could we parse in a way that we don't need to escape here? Or would this be a lot of work?

Reply:
That would be a lot of work. Philip's approach would get around this. Currently $>? also have to be escaped

Reply:
Ok, I see